Tests/Xpring/XpringClientTest.swift

```diff
@@ -41,7 +46,8 @@ final class XpringClientTest: XCTestCase {
       rawTransactionStatusValue: .success(fakeRawTransactionStatusValue),
       paymentHistoryValue: .success(fakePaymentHistoryValue),
       accountExistsValue: .success(fakeAccountExistsValue),
-      getPaymentValue: .success(fakeGetPaymentValue)
+      getPaymentValue: .success(fakeGetPaymentValue),
```

Comment:
Drive by observation: in the future we can maybe make these parameters optional or have a default?

Reply:
I agree - I was wondering how to handle this as it potentially grows larger and larger.  maybe can refactor all at once in future